### ISSUE TITLE
token not found. Cannot continue

### ISSUE THREAD

**Issue by moralespetit** [trocotronic/edistribucion#36]
Hi! 
seems since new edistribucion update I can't login and get the token. 
Any idea to fix it? 

Thanks in advance

![edistribucion_error](https://github.com/user-attachments/assets/e83a99a5-ff4a-4708-82ef-e93f96f0981d)

**Comment by dmachadol** [comment]
There is a new stage to confirm login and token. @trocotronic can you check this pls?.

**Comment by tasman09** [comment]
I have the same problem. Does anyone know how it can be fixed?

**Comment by moralespetit** [comment]
no idea. I was trying to debug and find a solution but I'm not a web expert

maybe @duhow  can take a look and help us :)

**Comment by trocotronic** [comment]
They introduced an authenticator proxy that sends a token to Salesforce. Then the user is registered by the backend and granted. 
The bad part is I don’t know how or where to grab that token. The JS code of this process is a monster and I didn’t succeed to find it or how to produce it.

**Comment by moralespetit** [comment]
@trocotronic can we do something to help ?

**Comment by trocotronic** [comment]
Not really. I have all the flowgraph but I don't know how to generate the token. If anyone knows how to generate the token in the Salesforce JS code, please tell it to me.

**Comment by gnomo22** [comment]
Hi all, not sure if correct, but I got the AuraToken by doing what follows:

```
logging.info('Accessing to frontdoor')
r = self.__get_url(jr['events'][0]['attributes']['values']['url'])
req_cookies = r.cookies.get_dict()
self.__fill_cookies(req_cookies)

r = self.__get_url('https://zonaprivada.edistribucion.com/areaprivada/salesforceIdentity/flowLightningOut.app?aura.format=JSON&aura.formatAdapter=LIGHTNING_OUT', cookies=self.__cookies)
jotason = r.json()
auraconfig = jotason.get('auraConfig')
self.__token = auraconfig.get('token')
```

Where the function ___fill_cookies_ is the following one:

```
def __fill_cookies(self, req_cookies: dict):
    self.__cookies['oid'] = req_cookies.get('oid')
    self.__cookies['sid'] = req_cookies.get('sid')
    self.__cookies['sid_Client'] = req_cookies.get('sid_Client')
    self.__cookies['clientSrc'] = req_cookies.get('clientSrc')
```

However, next step should be getting login info, thus getting the _account_id_ and _name_, but I could not find it. Hope it helps someone if you are trying to make it work. I will post if I finally make it work.

Cheers,
Adrián

**Comment by moralespetit** [comment]
![Image](https://github.com/user-attachments/assets/fb19256c-1619-4840-8db0-0fcd47cd2214)

The issue is checked and fixed.
Thank you so much  for the effort
